### ISSUE TITLE
BatchNorm momentum parameter in `model.py`

### ISSUE THREAD

**Issue by crlandsc** [frednam93/TAP-FDY-SED#2]
@frednam93 Thank you for sharing your great work on SED!

In line 515 within `model.py` (`cnn.add_module("batchnorm{0}".format(i), nn.BatchNorm2d(out_dim, eps=0.001, momentum=0.99))`), you have hardcoded the momentum as 0.99. This seems like it could be unstable based on my understanding of how PyTorch calculates the momentum for BatchNorm.

This would effectively use a coefficient of 0.99 for the current batch's statistics and 0.01 for the running average, nearly negating any stability benefits. The default is 0.1 (i.e., giving 90% weight to the running average), so I was curious why you chose such a high momentum.

Again, thanks for this open source work!

**Comment by frednam93** [comment]
Hi! Thanks for your interest in my work. For batch norm, I used it as it was from the DCASE2021 task4 baseline. So I did not consider experimenting on BN. But your point seems interesting. Let me know if you get to find out if your approach is better!

**Comment by crlandsc** [comment]
Thanks for the response. Ultimately, I ended up replacing it with GroupNorm (and often use it in place of BatchNorm). In my experience with smaller-scale models, batch sizes, & datasets, GroupNorm performs very similarly to BatchNorm and it's been more stable for me during training.
